src/unix/bsd/apple/mod.rs

```diff
@@ -22,6 +22,13 @@ pub type nl_item = ::c_int;
 pub type id_t = ::c_uint;
 pub type sem_t = ::c_int;
 pub type idtype_t = ::c_uint;
+cfg_if! {
```

Comment:
There are submodules to do the x86/x86_64 distinction, can this be pushed down into those submodules?